### DISCUSSION TITLE
Build xDripJH with fix to Libre2 Serial number issie

### DISCUSSION THREAD

**Discussion by micha102** [jakobheyman/xdripJH#2] [General]
Hello and thanks for this wonderful fork opitmized for Freestyle Libre sensors.
So, my friend changed her sensor this week but she suffers from the issue documented here :

https://github.com/NightscoutFoundation/xDrip/discussions/2845#discussioncomment-5995246

It's fixed in the main xDrip application. Could you please merge the fix ?
She doesn't want to use the xDrip master branch because xDripJH optimization are just wonderful.

Thank you so much in advance.

**Comment by jakobheyman** [comment]
Sorry for a very slow reply!
I did not check here for a long time and did not see your question until now.

I will try to update xdripJH and merge with all changes in the master branch but it might take some more time (hopefully not more than 1-2 weeks but I don't really know). I have been having problems with building the apk and that, in combination with too little time over to try to fix it, is the reason there has been no update for a long time. I will try to get it done in the next week but there is a risk that it could take 1, 2, 3... more weeks before I finally manage to fix it.

**Reply by micha102** [reply]
Thanks a lot Jakob, I really appreciate it.

**Reply by jakobheyman** [reply]
Hi again,
I have finally managed to build the apk and hopefully the issue above should now be fixed.
